src/components/Tokens/TokenTable/NetworkFilter.tsx

```diff
@@ -1,5 +1,7 @@
+import { ChainId } from '@uniswap/sdk-core'
 import Badge from 'components/Badge'
 import { getChainInfo } from 'constants/chainInfo'
+import { Chain } from 'graphql/data/__generated__/types-and-hooks'
 import {
   BACKEND_NOT_YET_SUPPORTED_CHAIN_IDS,
```

Comment:
It should be added to this array. `BACKEND_NOT_YET_SUPPORTED_CHAIN_IDS` is only used to filter out Chains which don't have Token Explore data. We could also make that more clear by renaming

Reply:
BACKEND_NOT_YET_SUPPORTED_CHAIN_IDS is also used to render the "search support coming soon" message in the search bar. 



<img width="500" alt="image" src="https://github.com/Uniswap/interface/assets/5773490/529de34e-4278-42ce-ae33-9cf60a721beb">